src/AccessibilityInsights.RulesTest/Library/BoundingRectangleContainedInParentTest.cs

```diff
@@ -294,5 +294,27 @@ public void BoundingRectangleContainedInParent_HorizontallyScrollablePass()
 
             Assert.AreEqual(Rule.Evaluate(e), EvaluationCode.Pass);
         }
+
+        [TestMethod]
+        public void BoundingRectangleContainedInParent_ChromeWindow_NotApplicable()
+        {
+            using (var e = new MockA11yElement())
```

Comment:
I wonder if we should use moq instead for new tests?

Reply:
discussed offline, we will consider refactoring MockA11yElement over time to internally use moq